code/process_sequences.sh

```diff
@@ -11,31 +11,28 @@
 # https://www.cyberciti.biz/faq/run-execute-sh-shell-script/
 # https://askubuntu.com/questions/420981/how-do-i-save-terminal-output-to-a-file
 
-FILE="/blast-db/sars-cov-2-seq-data/2020-10-15_ncbi_sars_cov_2_sequences.fasta.gz"
-
 if [ $# -eq 0 ]
 then
- echo "There is no file path provided"
+ echo "There is no file path provided."
```

Comment:
You should tell the user what to do to solve the problem. Something like: "This script takes the path to a compressed or uncompressed fasta file as input."

Reply:
Sure thing!